source/fundamentals/linq.txt

```diff
@@ -609,6 +608,35 @@ The following is a subset of the documents contained in the returned results:
    { "_id" : ObjectId(...), "name" : "Abaleh", "restaurant_id" : "50009096", "cuisine" : "Mediterranean", "address" : { ... }, "borough" : "Manhattan", "grades" : [...] }
    ...
 
+.. _csharp-linq-convert:
+
+$convert
+~~~~~~~~
+
+You can use the ``Mql.Convert()`` method in a ``Select()`` projection to
+convert a value from one type to a different specified type. To learn
+more about conversion behavior and permitted conversions, see the
+:manual:`$convert reference </reference/operator/aggregation/convert/>`
+in the {+mdb-server+} manual.
+
+The ``Convert()`` method takes the following parameters:
+
+- Value to convert.
+- ``ConvertOptions<Tto>`` instance that specifies the type to convert to
+  and options. Some conversions require you to specify certain options, but
+  you can also set options to handle errors or null values.
+
+The following code converts the ``RestaurantId`` string values to
+``int`` values in a projection by using LINQ:
+
+.. code-block:: csharp
+   
+   var query = queryableCollection
+       .Select(r => Mql.Convert(r.RestaurantId, new ConvertOptions<int>()));
```

Comment:
Hi @papafe would you prefer if we showed a more complex example that uses the options?

Reply:
I think the example is fine, it's difficult to make a more complex example using the dataset we have in this page. But at least you can show `OnError` or `OnNull`. So it could be:

```
   var query = queryableCollection
       .Select(r => Mql.Convert(r.RestaurantId, new ConvertOptions<int>( OnError = -1, OnNull = 0 )));
```